### PR TITLE
Adjust Time Throughout

### PR DESCRIPTION
**Pull request by tbosch82**
Adding dayjs to entire application and adding time stamps (in addition to date stamps already implemented) to all articles and charts. 

article
topTweets

### CODE REVIEW

**Comment by tbosch82** [comment]
@ericdudley Dayjs is integrated and I also added timestamps (per Mike's request) to articles and top tweets. Let me know what you think.

File: demo/src/util/store.ts [hunk 1]
```diff
@@ -358,22 +359,22 @@ class Store {
         id: getRandomId(),
         sources: [],
         typeLabel: "",
-        startTime: Date.now() - 1000 * 60 * 60 * 24 * 56,
-        endTime: Date.now(),
+        startTime: dayjs().subtract(30, "day"),
```
Comment (ericdudley) [inline demo/src/util/store.ts]: Originally I was thinking we would leave all of the times as `number` timestamps and just use a date library to do this type of time arithmetic.

Comment (ericdudley) [inline demo/src/util/store.ts]: Originally I was thinking we would leave all of the times as number timestamps and just use a date library to do this type of time arithmetic.

File: demo/src/util/store.ts [hunk 2]
```diff
@@ -500,20 +499,22 @@ class Store {
...
 
-  @action changeChartDataStartTime = (chart: IChart, dataStartTime: Date) => {
-    if (chart.dataEndTime > dataStartTime.getTime()) {
-      chart.dataStartTime = dataStartTime.getTime();
+  @action changeChartDataStartTime = (chart: IChart, dataStartTime: Dayjs) => {
+    if (dayjs(dataStartTime).isBefore(chart.dataEndTime)) {
+      chart.dataStartTime = dataStartTime;
       chart.sources.forEach(s => (s.startTime = chart.dataStartTime));
-      chart.startTime = Math.min(chart.startTime, chart.dataStartTime);
+      if (chart.dataStartTime.isBefore(chart.startTime))
```
Comment (ericdudley) [inline demo/src/util/store.ts]: These types of comparisons are a nice use also.

File: demo/src/util/util.ts [hunk 3]
```diff
@@ -18,11 +18,6 @@ export function stringArraysEqual(arr1: string[], arr2: string[]) {
   );
 }
 
-export function tsToDateString(ts: number) {
```
Comment (ericdudley) [inline demo/src/util/util.ts]: Have you verified that the date format from DateJS is compatible with the API and plotly?

Comment (tbosch82) [reply demo/src/util/util.ts]: This function shouldn't have been removed. I am adding it back in.

**Comment by tbosch82** [comment]
@ericdudley We can move everything back to numbers, I just figured rather than always converting back to unix it would be simpler to keep everything in the same format. Dayjs serializes to JSON in ISO8601 format, similar to how the native Date object works.

**Comment by ericdudley** [comment]
@tbosch82 Let's keep the type change then, should be fine.

**Pull request merged**